### PR TITLE
Release 0.18

### PR DESCRIPTION
**Pull request by sharksforarms**
release prep

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
 # Changelog
 
 ## [Unreleased]
+
+## [0.18.0] - 2024-08-05
+
+Usability
 - Add attributes `seek_from_start`, `seek_from_current`, `seek_from_end`, and `seek_rewind` to control the position of the reader before reading a field ([#360](https://github.com/sharksforarms/deku/pull/360))
```
Comment (wcampbell0x2a) [inline CHANGELOG.md:8]: This wasn't you, this was me. However, we should mention that `Seek` is now required.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+- Support added for unit structs ([#450](https://github.com/sharksforarms/deku/pull/450))
+
+Performance
+- The use of bits parsing is now an optional feature. If bit-level parsing is not usefull for your application, you can disable it to get some performance benefits. ([#446](https://github.com/sharksforarms/deku/pull/446))
```
Comment (wcampbell0x2a) [inline CHANGELOG.md:12]:
```suggestion
- The use of bits parsing is now an optional feature. If bit-level parsing is not useful for your application, you can disable it to get some performance benefits. ([#446](https://github.com/sharksforarms/deku/pull/446))
```

**Review by wcampbell0x2a — CHANGES_REQUESTED** [review]

**Comment by sharksforarms** [comment]
@wcampbell0x2a done

**Review by wcampbell0x2a — DISMISSED** [review]

**Comment by wcampbell0x2a** [comment]
I think https://github.com/sharksforarms/deku/pull/461 wasn't released last time either? Could deserve a mention.

**Pull request merged**